wal

```diff
@@ -196,6 +196,9 @@ set_wallpaper() {
             elif type -p habak >/dev/null; then
                 habak -mS "$wal"
 
+            elif (($(uname) == "Darwin")); then
```

Comment:
Add `os="$(uname)"` to the bottom of the "Internal Variables" section.

Then change the condition to use the following syntax.

```sh
elif [[ "$os" == "Darwin" ]]; then
```